app/grandchallenge/challenges/forms.py

```diff
@@ -189,9 +150,23 @@ def clean(self):
 )
 
 
-class ChallengeRequestForm(
-    ChallengeRequestBudgetFieldValidationMixin, forms.ModelForm
-):
+class ChallengeRequestForm(forms.ModelForm):
+    algorithm_selectable_gpu_type_choices = forms.MultipleChoiceField(
+        initial=get_default_gpu_type_choices(),
+        choices=[
+            (choice.value, choice.label)
+            for choice in [
+                GPUTypeChoices.NO_GPU,
+                GPUTypeChoices.T4,
+                GPUTypeChoices.A10G,
```

Comment:
Why only these 3?

Reply:
These are the only ones we want to enable right now. K80s - too old, V100s - overkill for the types of challenges we run right now, A100s - overkill and only available as multi-GPU instances.